FinSearchUnified/Tests/Bundle/StoreFrontBundle/Gateway/Findologic/CustomFacetGatewayTest.php

```diff
@@ -24,7 +30,7 @@ public function tearDown()
     }
 
     /**
-     * @throws Exception
+     * @throws \Exception
```

Comment:
Since you already touched this:
* Import class `Exception`.
* Use `@throws Exception` instead.